doc/specs/#658 - WinGet Download.md

```diff
@@ -0,0 +1,209 @@
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Starting package download...
+   \
+Successfully verified installer hash
+Successfully downloaded
+
+PS: C:\> 
+```
+
+When the user runs the Help action on the WinGet Download command, the following information will be provided:
+
+```
+PS C:\> WinGet Download --help
+Windows Package Manager v1.1.1
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Downloads an identified package's installer, either found by searching a configured source or directly from a manifest. By default, the query must case-insensitively match the id, name, or moniker of the package. Other fields can be used by passing their appropriate option.
+
+The following command alias are available
+
+The following arguments are available:
+
+The following options are available
+    --id                            Filter results by id 
+    --name                          Filter results by name
+    --moniker                       Filter results by moniker
+    --scope                         Select install scope (user or machine)
+    --locale                        Locale to use (BCP47 format)
+    --filename                      The value to rename the downloaded executable file to
+    -v, --version                   Use the specified version; default is the latest version
+    -s, --source                    Find package using the specified source
+    -a, --architecture              Select the architecture to install
+    -e, --exact                     Find package using exact match
+    -o, --output                    Location to download file(s) to
+    --installer-type                Specifies the filetype for the installer
+    --ignore-security-hash          Ignore the installer hash check failure
+    --accept-package-agreements     Accept all license agreements for packages
+    --help                          Shows help about the selected command
+    --wait                          |Prompts the user to press any key before existing
+    --verbose, --verbose-logs       Enables verbose logging for WinGet
+
+```
+
+WinGet search will be used to enlighten customers of the package's installers downloadable status. the new column will only be displayed if the application has an installer can be downloaded.
+
+```
+PS C:\> winget search Microsoft.VisualStudioCode
+Name                                  Id                                  Version Source Download
+-------------------------------------------------------------------------------------------------
+Microsoft Visual Studio Code          Microsoft.VisualStudioCode          1.75.0  winget True
+Microsoft Visual Studio Code Insiders Microsoft.VisualStudioCode.Insiders 1.76.0  winget
+```
+
+### WinGet Setting - Default Download Output
+
+A new setting will be included as the default output location for the user when package's installer is downloaded.
+
+```PS
+PS C:\> winget settings export
```

Comment:
Default should be a user setting, not an admin setting, and so wouldn’t be included in export